### PR TITLE
Add EIP: Dual Layer Token

### PR DESCRIPTION
**Pull request by Aboudjem**
EIP-6960, which introduces the Dual Layer Token Standard for better token classification and management on Ethereum. This standard is designed to address limitations in the current ERC-1155 standard when dealing with tokens that have multiple classifications.

- Ethereum-magicians discussion: [EIP-6960-dual-layer-token](https://ethereum-magicians.org/t/eip-6960-dual-layer-token/14070)
- Full documentation here: [Dual Layer Token - Gitbook](https://polytrade.gitbook.io/rwa-marketplace/upcoming-features/dlt-token-standard)
- full implementation available here https://github.com/polytrade-finance/dual-layer-token

### CODE REVIEW

File: EIPS/eip-6960.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+This EIP strives to overcome this limitation by proposing a token standard incorporating a dual-layer classification system, allowing for enhanced organization and management of tokens, especially in situations where additional sub-categorization of token types is necessary.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### DLT Interface
+
+```solidity
+// SPDX-License-Identifier: MIT
```
Comment (SamWilsn) [inline EIPS/eip-6960.md:40]: Code inline in the proposal must be licensed under `CC0-1.0`. Code in the assets directory can be MIT if you'd like.

File: EIPS/eip-6960.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+/**
+ * @title DLT token standard interface
+ * @dev Interface for any contract that wants to implement the DLT standard
+ */
+interface IDLT {
+    struct Balance {
+        uint256 mainBalance;
+        mapping(uint256 => uint256) subBalances;
+    }
```
Comment (SamWilsn) [inline EIPS/eip-6960.md:51]: Is this struct used in the interface itself?

File: EIPS/eip-6960.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+     * See {setApprovalForAll}
+     */
+    function isApprovedForAll(
+        address owner,
+        address operator
+    ) external view returns (bool);
+}
+```
+
+### DLTReceiver Interface
```
Comment (SamWilsn) [inline EIPS/eip-6960.md:252]: Code in headers should be enclosed in backticks (\`)

File: EIPS/eip-6960.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+- Should we extend the ERC-1155 standard?
+As the ERC-1155 standard is not designed to support a layered classification and requires significant modifications to do so, we concluded that it would not be appropriate to extend it for the dual-layer token standard. Hence, a standalone implementation would be a more suitable approach.
+
+## Backwards Compatibility
+
+No backward compatibility issues found.
+
+## Test Cases
+
+The standard is implemented and tested with full coverage on Polytrade's Dual-Layer-Token repository
```
Comment (SamWilsn) [inline EIPS/eip-6960.md:320]: Please remove this external link (even if it isn't an HTML/markdown link :wink:)

Comment (SamWilsn) [reply EIPS/eip-6960.md:321]: Sorry if I wasn't clear. Mentioning your repository for tests cases or a reference implementation isn't permitted, as it violates our external link policy. Either include your test cases and implementation in the EIP itself, or in the assets directory.

**Review by SamWilsn — CHANGES_REQUESTED** [review]

File: EIPS/eip-6960.md (hunk 4 above)
Comment (Aboudjem) [reply EIPS/eip-6960.md]: My bad, I forgot about that one. Thanks for reminding me. It's all fixed now!

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**